OperatorAssignment.js

```diff
@@ -0,0 +1,57 @@
+// **********************************  JavaScript Task #2 Operator Problem  ***********************************************
+
+const array  = ["OR",["<","a","b"],["AND",["==","c","d"],["!=","e","f"]]]
+
+const flatedArray = array.flat(Infinity); //removing nested arrays
+let length = flatedArray.length;
+let i = length-1;
+console.log('\n');
+let bufferArray=[];
+while( i >= 0 ){
+    if( isOperator(flatedArray[i]) === true ){           // inCase of operator pop two operands and place operator between them and push into stack         
+        let firstOperand = bufferArray[bufferArray.length -1];
+        bufferArray.pop();
```

Comment:
You can write it as:
- let firstOperand = bufferArray.pop()
- let secondOperand = bufferArray.pop()